install.rb

```diff
@@ -255,10 +255,13 @@ def install_binfile(from, op_file, target)
       tmp_file2 = Tempfile.new('hiera-wrapper')
       cwv = <<-EOS
 @echo off
-setlocal
-set RUBY_BIN=%~dp0
-set RUBY_BIN=%RUBY_BIN:\\=/%
-"%RUBY_BIN%ruby.exe" -x "%RUBY_BIN%hiera" %*
+SETLOCAL
+if exist "%~dp0environment.bat" (
+  call "%~dp0environment.bat" %0 %*
+) else (
+  SET PATH=%~dp0;%PATH%
+)
+ruby.exe -S -- hiera %*
```

Comment:
@melissa Does it matter that the default search order for executables is to look in the current working directory and THEN the path?  Conceivably it could be possible to have a different ruby.exe in the local dir but I think this might be too unlikely to happen.

What do you think?